application-templates/starter/src/components/channel-details/messages.js

```diff
@@ -0,0 +1,51 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  backToChannelsList: {
+    id: 'ChannelDetails.backToChannelsList',
+    description: 'Back to channels list text',
```

Comment:
Nit: I would omit all `description` fields, they are not really relevant for this case.